java/src/main/java/org/whispersystems/signalservice/loki/api/LokiDotNetAPI.kt

```diff
@@ -23,6 +23,15 @@ import java.io.IOException
 import java.io.InputStream
 import java.util.*
 
+object RequestCache {
```

Comment:
Seems a bit too complex, should remove it in favour of a simple private hashmap?
Thoughts?